res/database/Default/database_programmes.xml

```diff
@@ -35776,6 +35886,7 @@ Zawiera wiele kalamburów i [3|Full] w nagrodzonym Oscarem humorze.</pl>
 			<title>
 				<de>[0|First] - Der Extraterr-Friesische</de>
 				<en>[0|First] - The Extraterr-Frisian</en>
+				<pl>[0|First] - Der Extraterr-Friesische</pl>
```

Comment:
German version intentional?

Reply:
A German film is a German title when there is no Polish equivalent.